docs/samples/Microsoft.ML.Samples/Dynamic/ModelOperations/SaveLoadModel.cs

```diff
@@ -35,11 +36,15 @@ public static void Example()
                 model = mlContext.Model.Load(file, out DataViewSchema schema);
 
             // Create a prediction engine from the model for feeding new data.
-            var engine = mlContext.Model.CreatePredictionEngine<Data, Transformation>(model);
+            var engine = mlContext.Model.CreatePredictionEngine<Data,
```

Comment:
@eerhardt Is it ok to break generics like this or should all of it be in the same line as its corresponding class?

Reply:
CC: @natke

Reply:
I'll fix it so that I split before the "<" in a later pull request. Thanks!